### PR TITLE
Added shared secret and some result properties to apple iap handling

### PR DESCRIPTION
**Pull request by HolgerFrank**

### CODE REVIEW

**Comment by ronkorving** [comment]
Thanks! :)

**Comment by ronkorving** [comment]
@AlmirKadric mind reviewing too?

File: lib/apple/index.js [hunk 1]
```diff
@@ -35,7 +35,12 @@ function parseResult(result) {
 	var receipt = result.receipt;
 
 	return {
+		status: result.status,
```
Comment (AlmirKadric) [inline lib/apple/index.js]: my only comment would be that these additional return values are documented in the Readme.md file

Comment (ronkorving) [inline lib/apple/index.js]: I have to agree that that's a bit against the original philosophy (where properties at the top level are universal, regardless of which store is being used). We can either change the philosophy to say "at least properties A, B, C, optionally augmented with store specific props", or we put all of these in another sub-object.

@HolgerFrank any input on this?

**Comment by ronkorving** [comment]
Any updates?

**Comment by jerolimov** [comment]
Need this too :+1:

**Comment by KLVTZ** [comment]
@ronkorving let's close this in favor of #21

**Comment by ronkorving** [comment]
Resolved by #23

**Pull request closed without merging**